universal-application-tool-0.0.1/app/services/applicant/question/StaticContentQuestion.java

```diff
@@ -0,0 +1,66 @@
+package services.applicant.question;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import services.Path;
+import services.applicant.ValidationErrorMessage;
+import services.question.types.QuestionType;
+
+public class StaticContentQuestion implements PresentsErrors {
```

Comment:
question: Does this one need to implement `PresentsErrors`? Can't remember how necessary this is. If it's easy to avoid, let's do it, but I get it if it's really inconvenient to remove this.

Reply:
`isAnswered` comes from `PresentsErrors` so we at least need to implement it from that.

Reply:
It needs to be answered so applicants don't get stuck on a block with static content.